fizzbuzz.js

```diff
@@ -5,43 +5,46 @@ For the multiples of five print “Buzz”
 For numbers which are multiples of both three and five print “FizzBuzz”."
 */
 
+// firts solution:
 const fizzbuzzAlgoritmus1 = () => {
     let numbers = [];
+    // the value is set to 50 just to distinguish the result from the second solution (when run on the web)
     for (let i = 1; i <= 50; i++) {
         if (i % 3) {
-            // pokud neni delitelne 3 - NE
+            // NOT a multiply of 3
             if (i % 5) {
-                // pokud neni delitelne 5 - NE
+                // NOT a multiply of 5
                 numbers.push(i);
             } else {
-                // pokud je delitelne 5 - ANO
+                // IS a multiply of 5
                 numbers.push("Buzz");
             }
         } else {
-            // pokud je delitelne 3 - ANO
+            // IS a multiply of 3
             if (i % 5) {
-                // neni delitelne 5 - NE
+                // NOT a multiply of 5
                 numbers.push("Fizz");
             } else {
-                // je delitelne i 5 - ANO
+                // IS a multiply of 5
                 numbers.push("FizzBuzz");
             }
         }
     }
     return numbers;
 };
 
+// second solution / simplification:
 const fizzbuzzAlgoritmus2 = () => {
     let numbers = [];
     for (let i = 1; i <= 100; i++) {
         let result = "";
-        // pokud je delitelne 3
+        // IS a multiply of 3
         if (i % 3 === 0) {
             result += "Fizz";
-        } // pokud je delitelne 5
+        } // IS a multiply of 5
         if (i % 5 === 0) {
             result += "Buzz";
-        }
+        } // did not followed any of the previous conditions
```

Comment:
better english
```suggestion
        } // did not follow any of the previous conditions
```
or even clearer
```suggestion
        } // NOT a multiply of 3 nor 5
```